swagger.yaml

```diff
@@ -3092,6 +3092,32 @@ definitions:
               format: uuid
             role:
               $ref: '#/definitions/host-role-update-params'
+      hosts_disks:
+        type: array
+        x-go-custom-tag: gorm:"type:varchar(64)[]"
+        description: The desired boot disk for hosts associated with the cluster.
+        x-nullable: true
+        items:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
```

Comment:
Disks have UUID's? Where are they defined?

Reply:
this is the host-id section. all the parameters follow the pattern of list of tupples [(host-id, param), ...]

Reply:
Right, got it

Reply:
This is the host ID right?  Where does the user specify the disk?